.github/CODEOWNERS

```diff
@@ -350,6 +350,11 @@
 /x-pack/plugins/timelines @elastic/security-threat-hunting
 /x-pack/test/case_api_integration @elastic/security-threat-hunting
 /x-pack/plugins/lists @elastic/security-detections-response
+/x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps @elastic/security-onboarding-and-lifecycle-mgt
```

Comment:
From our conversation, here are a few more paths we should consider:

```
x-pack/plugins/security_solution/public/management
x-pack/plugins/security_solution/common/endpoint
x-pack/plugins/security_solution/public/common/lib/endpoint* 
x-pack/plugins/security_solution/public/common/components/endpoint
x-pack/plugins/security_solution/server/endpoint
x-pack/plugins/security_solution/server/lib/license
x-pack/plugins/security_solution/server/fleet_integration
x-pack/plugins/security_solution/scripts/endpoint/event_filters
x-pack/plugins/security_solution/scripts/endpoint/trusted_apps
x-pack/test/security_solution_endpoint/apps/endpoint
(or maybe all of x-pack/test/security_solution_endpoint. since we also own almost all of the other code here (with exception of endpoint_telemetry)
x-pack/test/security_solution_endpoint_api_int/[metadata*, package, policy]
```